# Performs validity check on anchor hrefs. Warns when anchors are used as buttons. (anchor-is-valid)

Please describe the origin of the rule here.

## Rule Details

This rule aims to...

Examples of **incorrect** code for this rule:

```js
// fill me in
```

Examples of **correct** code for this rule:

```js
// fill me in
```

### Options

If there are any options, describe them here. Otherwise, delete this section.

```js
{
  "rules": {
    "jsx-a11y/anchor-is-valid": [ "error", {
      "aspects": [ "noHref", "invalidHref", "preferButton" ]
    }]
  }
}
```

## When Not To Use It

Give a short description of when it would be appropriate to turn off this rule.

## Further Reading

If there are other links that describe the issue this rule addresses, please include them here in a bulleted list.
